whitepaper/chapter01.md

```diff
@@ -0,0 +1,140 @@
+
+In cloud native systems, the atomic unit of deployment is the container, on an immutable and decoupled compute environment that nearly completely abstracts function from the underlying physical infrastructure and network architecture. At present, the most commonly used container environment is an OCI-compliant implementation. A container is different than a Virtual Machine (VM), which was the initial base unit of cloud computing. A VM hosts a Guest OS capable of delivering a service's required compute and networking. This guest runs on top of a hypervisor consuming the virtual infrastructure and orchestration it provides. In contrast, containers run on top of a shared Host OS (with a shared kernel) kernel. This distinction is key because containerisation enables immutability, disposability and resource efficiency in ways that are not easily achievable with VMs.
+
+For Kubernetes specifically, the atomic unit of management is a Pod. A Pod is a set of application containers which are 
+co-located and co-scheduled on the same machine (Node in Kubernetes terms) which run with a shared context and the same Linux networking namespace. For telco (and other) cloud native applications, the Pod is the basic building block of deployment, scaling operations, and upgrades.
+
+Cloud native applications consist of sets of cloud native microservices. The configuration of these microservices is not changed after deployment (note, this does allow for changes in application data such as "add a new user"). New features for a cloud native application and the underlying Pods are rolled out as new artifacts and new configuration. Cloud native systems are configured declaratively. This means that the system configuration describes “what” a loosely coupled system should look like, not ”how” the system should be created. The “how” of the application is determined by the tooling (e.g. the orchestrator, operators, and CRDs).
+
+<a name="1.4"></a>
+## 1.4 Cloud Native Network Functions
+
+A cloud native network function (CNF) is a cloud native application that implements or facilitates network functionality. A cloud native network function consists of one or more microservices, and has been developed using Cloud Native Principles including immutable infrastructure, declarative APIs and a “repeatable deployment process”.
+
+An example of a simple CNF is a packet filter which implements a single piece of network functionality as a microservice. A firewall is an example of a CNF which may be composed of more than one microservice.
```

Comment:
I have lost count on how many times we restate that a CNF is cloud native application that ... before submitting, we should revalidate how much we re-express the definition.
As for an exemple of a good CNF model, I always liked the UTM example. It would be made of a set of microservices per each behaviour (Packet-IO, FW, DPI, NAT, MGMT-API, StateDB) linked together either in "pipeline" or "snake-mode" but still compose a full UTM service